### ISSUE TITLE
0.109 has deprecation warning about .translations folder

### ISSUE THREAD

**Issue by mdegat01** [pinkywafer/Anniversaries#53]
Upgraded to 0.109 and this issue showed up in my log:
```
 Log Details (WARNING)

Logger: homeassistant.helpers.translation
Source: helpers/translation.py:206
First occurred: 3:56:35 PM (10 occurrences)
Last logged: 3:56:40 PM

• anniversaries: the '.translations' directory has been moved, the new name is 'translations', starting with Home Assistant 0.111 your translations will no longer load if you do not move/rename this 
```

**Comment by pinkywafer** [comment]
This has been resolved by #48.
will be in the next release

**Comment by pinkywafer** [comment]
closed by #48 
released in 2.2.0
